config/default.js

```diff
@@ -54,21 +54,39 @@ module.exports = {
     'isDevelopment',
   ],
 
-  // Content security policy.
+  // Content Security Policy.
+  // NOTE: This config should be overridden on a per app basis
+  // if you're not updating the config for all apps.
+  // NOTE: if a config contains a var, it must be set
+  // for all overriding configs.
   CSP: {
     directives: {
-      defaultSrc: ["'self'"],
+      defaultSrc: ["'none'"],
+      baseUri: ["'self'"],
+      childSrc: ["'none'"],
       connectSrc: [
         "'self'",
         apiHost,
       ],
+      formAction: ["'self'"],
+      frameSrc: ["'none'"],
+      frameAncestors: ["'none'"],
```

Comment:
This is locking down what can be a frame ancestor. We'll need to configure this for the disco pane later on.